code/game/objects/structures/false_walls.dm

```diff
@@ -83,28 +83,38 @@
 		qdel(src)
 	return T
 
+/obj/structure/falsewall/tool_act(mob/living/user, obj/item/tool)
+	if(opening)
+		to_chat(user, span_warning("You must wait until the door has stopped moving!"))
+		return TRUE
+	return ..()
+
+/obj/structure/falsewall/screwdriver_act(mob/living/user, obj/item/tool)
+	. = TRUE
+	if(density)
+		var/turf/T = get_turf(src)
+		if(T.density)
+			to_chat(user, span_warning("[src] is blocked!"))
+			return
+		if(!isfloorturf(T))
```

Comment:
```suggestion
		var/turf/loc_truf = get_turf(src)
		if(loc_truf.density)
			to_chat(user, span_warning("[src] is blocked!"))
			return
		if(!isfloorturf(loc_truf))
```

Reply:
Fixed in https://github.com/tgstation/tgstation/pull/64428/commits/7a814ca8d8a41041a237da7db94c75e2cced2643